xrviz/dashboard.py

```diff
@@ -122,7 +156,8 @@ def create_indexed_graph(self, *args):
         dims_to_agg = self.kwargs['dims_to_agg']
         graph_opts = {'x': x,
                       'y': y,
-                      'title': self.var}
+                      'title': self.var,
+                      'cmap': 'Inferno'}
```

Comment:
Why the absolute cmap value here?

Reply:
`cmap` option is present in style panel. It would be replaced once style panel is merged (while resolving conflicts).